RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -56,14 +50,45 @@ public class PaywallViewController: UIViewController {
     /// `Offerings.current` will be used by default.
     /// - Parameter fonts: An optional ``PaywallFontProvider``.
     /// - Parameter displayCloseButton: Set this to `true` to automatically include a close button.
-    public init(
+    public convenience init(
```

Comment:
just a question I have, there's only one `@objc` annotated method in this file, but we don't use it in phc, because it's written in Swift... is it exposed for folks that are using objective c? If so, should we expose another constructor?

Reply:
Also in the docs of this constructor it says "an optional PaywallFontProvider", but it's not optional

Reply:
> is it exposed for folks that are using objective c?

Yeah that's correct, because `PaywallFontProvider` isn't available in Swift.
I'll add another one with offering identifier for ObjC

Reply:
And fixed the docs, thanks!

Reply:
Actually I think I'm not going to do that. The ideal way is passing an `Offering` to minimize the chances that you'll pass something that won't be found. The `offeringIdentifier` one is just for us for the hybrids. Other users should pass `Offering`s